centos_ci.sh

```diff
@@ -133,7 +133,8 @@ function run_tests() {
   # In Jenkins slave we have pull secret file in the $HOME/payload/crc_pull_secret
   # this is copied over using https://github.com/minishift/minishift-ci-jobs/blob/master/minishift-ci-index.yaml#L99
   export PULL_SECRET_FILE=--pull-secret-file=$HOME/payload/crc_pull_secret
-  export BUNDLE_LOCATION=--bundle-location=$HOME/Downloads/$BUNDLE 
+  export BUNDLE_LOCATION=--bundle-location=$HOME/Downloads/$BUNDLE
```

Comment:
> Since bundle is not embeded so it is easy to do make release and test the release bits instead intermidate one.

I'd change the log to something like this:

-> Since the bundle is no longer embedded, `make release` can be used so that the CI tests builds similar to the ones which would get released